tests/src/Unit/AvPortalResourceTest.php

```diff
@@ -317,6 +317,28 @@ public function titleResourceDataProvider(): array {
         'langcode' => 'FR',
         'expected_title' => 'Italian title with markup, encoded \'characters\' & letters čö&įię.',
       ],
+      'langcode specified / existing title had more than 255 symbols' => [
+        'data' => [
+          'ref' => 'P-047441/00-05',
+          'titles_json' => [
+            'EN' => 'Press conference of Margrethe Vestager, Executive Vice-President of the European Commission, on an antitrust case: Commission fined Teva and Cephalon for delaying entry of cheaper generic medicine',
+            'FR' => 'Conférence de presse de Margrethe Vestager, vice-présidente exécutive de la Commission européenne, sur un cas de pratique anti-concurrentielle : la Commission a infligé des amendes à Teva et à Cephalon pour avoir retardé l&#39;entrée sur le marché d&#39;un médicament générique moins cher',
+          ],
+        ],
+        'langcode' => 'FR',
+        'expected_title' => 'Conférence de presse de Margrethe Vestager, vice-présidente exécutive de la Commission européenne, sur un cas de pratique anti-concurrentielle : la Commission a infligé des amendes à Teva et à Cephalon pour avoir retardé l\'entrée sur le marché d\'un…',
+      ],
+      'langcode specified / existing title had more than 255 symbols / after trim without ellipses we have 255 symbols' => [
```

Comment:
I am not sure about this test, it should be covering that after the trim (truncate) there's no ellipsis if we have 255 characters?